### PR TITLE
Doc-537: Review Partition

### PR DESCRIPTION
**Pull request by diana-macrometa**
- Broke partion.md into separate parts, placed in folder.
- Minor format and grammar edits.
- Added links to new topics to Query Guide index.

@StoyanTc - An example seems to be missing from purge-partition.md. Can you fill it in?

### CODE REVIEW

File: docs/cep/query-guide/partition/purge-partition.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    insert into <output stream>
+    select <attribute name>, <attribute name>, ...
+    from <input stream> ...
+end;
+```
+
+When using purge with an aggregation, use a `WITH()` property instead. For example:
+
+```
+
```
Comment (diana-macrometa) [inline docs/cep/query-guide/partition/purge-partition.md]: @StoyanTc, can you fill in the missing example? Or should I just delete lines 26 through 30?

Comment (StoyanTc) [reply docs/cep/query-guide/partition/purge-partition.md]: @diana-macrometa - This document is only for partitions why we should add example with `aggregation`?

Comment (diana-macrometa) [reply docs/cep/query-guide/partition/purge-partition.md]: This is how the document was when I started editing it, I did not add anything, just am wondering what to do with this empty example. Should I just delete it?

Comment (StoyanTc) [reply docs/cep/query-guide/partition/purge-partition.md]: Yes, I think we need to remove the line containing `aggregation`.  This seems to be a `copy/paste` from somewhere else.

Comment (diana-macrometa) [reply docs/cep/query-guide/partition/purge-partition.md]: Ready for review.

**Review by StoyanTc — APPROVED** [review]

**Pull request merged**